docs/index.md

```diff
@@ -162,8 +162,45 @@ When an issuer generates a new key to sign Health Cards, the public key SHALL be
 issuer's JWK set in its `jwks.json` file. Retired private keys that are no longer used to sign Health Cards SHALL be destroyed.
 Older public key entries that are needed to validate previously
 signed Health Cards SHALL remain in the JWK set for as long as the corresponding Health Cards
-are clinically relevant. However, if a private signing key is compromised, then the issuer SHALL immediately remove the corresponding public key
-from the JWK set in its `jwks.json` file and request revocation of all X.509 certificates bound to that public key.
+are clinically relevant. However, if a private signing key is compromised, then the issuer SHALL immediately remove the corresponding public key from the JWK set in its `jwks.json` file and request revocation of all X.509 certificates bound to that public key; verifiers will from then on reject all Health Cards signed using that key.
+
+### Revocation
+
+Individual Health Cards MAY be revoked using a revocation identifier property `rid` encoded in the `vc` claim of the JWT. This should be a short identifier, meaningless to the verifiers; the only constraint is that the identifier SHALL use the base64url alphabet (but doesn’t need to be base64url encoded). Issuers MAY use application-specific user identifiers for this purpose, but since these could be publicly listed in revocation lists, issuers SHOULD use a one-way transformation of the data combined with enough entropy to prevent reversal. It is RECOMMENDED to use the base64url encoding of the first 64 bits of the output of HMAC-SHA-256 (as specified in [RFC 4868](https://tools.ietf.org/html/rfc4868)) using a 256-bit random secret key concatenated with the `<<kid>>`.
```

Comment:
Can we impose an absolute upper bound of how long a `rid` can be?

Reply:
> Can we impose an absolute upper bound of how long a `rid` can be?

There is a natural goal of keeping these small to 1) still fit in v22 QR, and 2) keep the CRLs small, but we could define an upper bound if we need to enforce this. What would you suggest?

Reply:
Yeah, agree with the implicit goal. So I presume it only makes sense to have an absolute upper limit. I'm not sure what a good value is, should we aim to allow 256 bits, even though we recommend taking the first 64 only? So, in base64, that's like 43 chars, can we limit to 50 chars? Too much? Too little?

Reply:
OTOH should we just specify _how_ to calculate the `rid` and be done with it? I'm not sure what the risk of conflict is if we just take the first 64 bits.

Reply:
64 bit is safe to avoid accidental collisions. The only "attacker" in this case would be the issuer itself, who could try to generate collisions, a pointless exercice. Since this does not need to be processed by wallets and verifiers, I don't think we need/should enforce an algorithm; but the recommended hmac approach is a good way to transform an ID meaningful to the issuer into an opaque one to everyone else.

Reply:
Ok, but if we think 64 bits are enough, which is like ~11 base64 chars, then we can set a much lower limit?